template-processing/knotx/conf/knots/templateEngineKnot.conf

```diff
@@ -0,0 +1,25 @@
+# Vert.x event bus delivery options used when communicating with other verticles
```

Comment:
I'm wondering...
Maybe we should put here only the minimum of configs that are required to run the example. And remove all the rest, just to enable people to focus on what is important in that example.
@tomaszmichalak wdyt?

Reply:
agree